src/lib.rs

```diff
@@ -471,20 +471,20 @@ impl OffersMessageHandler for OfferHandler {
         responder: Option<Responder>,
     ) -> Option<(OffersMessage, ResponseInstruction)> {
         match message {
-            OffersMessage::InvoiceRequest(_) => {
-                log::error!("Invoice request received, payment not yet supported.");
-                None
-            }
+            OffersMessage::InvoiceRequest(_) => None,
             OffersMessage::Invoice(invoice) => {
-                info!("Received an invoice: {invoice:?}");
                 let secp_ctx = &Secp256k1::new();
-                let offer_context = context.unwrap();
+                let offer_context = match context {
```

Comment:
I think this block can be simplified to `let offer_context = context?;`

Reply:
Also can you split this separate change removing the unwrap into it's own commit?